conformance/lang/expressions/shift-expression/right_shift.balt

```diff
@@ -0,0 +1,4537 @@
+Labels: byte, int, int:Signed32, int:Signed8, int:Unsigned8, nil-literal, optional-type, shift-expr,
+        unary-minus, value:toBalString
+
+int i1 = -1;
+int? i2 = 7;
+int? i3 = ();
+
+int:Unsigned8 s1 = 255;
+int:Signed16? s2 = 2;
+int:Unsigned32? s3 = ();
+
+byte? b1 = 2;
+byte? b2 = ();
+
+function init() {
+    io:println(i2 >> i1); // @output 0
+    io:println(s1 >> i2); // @output 1
+    io:println(i2 >> s2); // @output 1
+    io:println(s2 >> b1); // @output 0
+    io:println(b1 >> i2); // @output 0
+
+    io:println((i3 >> s1).toBalString()); // @output ()
+    io:println((s2 >> s3).toBalString()); // @output ()
+    io:println((b2 >> i2).toBalString()); // @output ()
+    io:println((s3 >> i3).toBalString()); // @output ()
+    io:println((i3 >> b2).toBalString()); // @output ()
+}
+
+Test-Case: output
+Description: Test the '>>' expression at module level.
+Labels: byte, int, int:Signed16, int:Signed32, int:Signed8, int:Unsigned16, int:Unsigned32,
+        int:Unsigned8, shift-expr, unary-minus
+
+int i1 = 1221112;
+int i2 = 3;
+
+int:Signed8 s1 = 127;
+int:Unsigned8 s2 = 5;
+int:Signed16 s3 = -8;
+int:Unsigned16 s4 = 3;
+int:Signed32 s5 = -9;
+int:Unsigned32 s6 = 17;
+
+byte b1 = 255;
+byte b2 = 4;
+
+int r1 = i1 >> i2;
+
+int r2 = s1 >> s2;
+int r3 = s3 >> s4;
+int r4 = s5 >> s6;
+int r5 = i1 >> s1;
+int r6 = s6 >> i2;
+
+int r7 = b1 >> b2;
+int r8 = b1 >> i2;
+int r9 = i1 >> b2;
+int r10 = s1 >> b2;
+int r11 = b1 >> s4;
+
+int r12 = 12331321 >> 3;
+
+function init() {
+    io:println(r1); // @output 152639
+    io:println(r2); // @output 3
+    io:println(r3); // @output -1
+    io:println(r4); // @output -1
+    io:println(r5); // @output 0
+    io:println(r6); // @output 2
+    io:println(r7); // @output 15
+    io:println(r8); // @output 31
+    io:println(r9); // @output 76319
+    io:println(r10); // @output 7
+    io:println(r11); // @output 31
+    io:println(r12); // @output 1541415
+}
+
+Test-Case: output
+Description: Test '>>' expression with field access as operands.
+Labels: DecimalNumber, field-access-expr, int, mapping-constructor-expr, module-type-defn, record-type, shift-expr,
+        unary-minus
+
+function init() {
+    Record a = {x: 12};
+    Record b = {x: -2132};
+
+    int c = b.x >> 1;
+    io:println(c); // @output -1066
+    int d = 4000000 >> a.x;
+    io:println(d); // @output 976
+}
+
+type Record record {
+    int x;
+};
+
+Test-Case: output
+Description: Test '>>' expression with member access as operands.
+Labels: array-type, DecimalNumber, int, list-constructor-expr, member-access-expr, shift-expr
+
+function init() {
+    int[] b = [3, 40];
+
+    int c = 10 >> b[0];
+    io:println(c); // @output 1
+    int d = b[1] >> 3;
+    io:println(d); // @output 5
+}
+
+Test-Case: output
+Description: Test '>>' expression with function call as operands.
+Labels: DecimalNumber, function-call-expr, function-defn, int, shift-expr
```

Comment:
Need `return-stmt` label.

Reply:
It belongs to minimal subset. Therefore, we can ignore that label.